### DISCUSSION TITLE
Support Linux and MacOS packages (Become a Windows, Linux and MacOS package manager)

### DISCUSSION THREAD

**Discussion by trymeouteh** [ScoopInstaller/Scoop#6067] [Ideas]
Scoop is a simple to use, decentralized with the use of buckets, its easy to host a bucket using a git repo, and offer version management making it easy to downgrade a package if needed.

Please consider adding support for Linux and MacOS apps. I know there are many package managers for Linux, but scoop could be a Linux package manager that...

- Works on all (or most distros) and not just on Debian/Ubuntu or Arch or Fedora
- Offers version management on packages
- Decentralized because of buckets
- Already easy to host a bucket!

Pros and cons of existing linux package managers...

- Apt: 👍Bare metal installs, 👎Debian/Ubuntu only
- Pacman: 👍Bare metal installs, 👎Arch only
- AUR: 👎Arch only, 👎not decentralized
- Flatpaks: 👎Container apps (Not bare metal)
- Nix: 👍Works on most distros, 👎hard to understand, 👎not decentralized

If scoop was available for Linux, it will be my go to package manager without a doubt.

**Comment by HUMORCE** [comment]
I think you're looking for [Conda](https://github.com/conda/conda).

**Reply by trymeouteh** [reply]
Conda looks like a package manager for Python packages. Scoop is a package manager for applications.

**Reply by HUMORCE** [reply]
Conda is a generic package manager, not just for python packages. For example, github-cli(gh) has been distributed to multi platforms through it. https://github.com/cli/cli#installation

**Reply by edwinjhlee** [reply]
You can try pixi, asdf, pkgx. They are all cross-platform generic package manager. 

I personaly recommend using pixi, which is built upon conda ecosystem.

---

You can try [x-cmd](https://x-cmd.com). x-cmd has a small [pkg](https://x-cmd.com/pkg) manager of common binary tools. 
You can easily install pixi, asdf, pkgx using x-cmd. 

x-cmd also provide some enhance shell wrappers for scoop. Actually x-cmd enhances all of these package mangers in the same way.

<img width="843" alt="image" src="https://github.com/user-attachments/assets/901d90c0-0a1d-4708-a924-30d0515b3f93">

**Comment by josepheiba** [comment]
that would be great
